paws_collector.js

```diff
@@ -450,6 +454,14 @@ class PawsCollector extends AlAwsCollector {
         this.reportDDMetric('api_throttling', 1)
         return cloudwatch.putMetricData(params, callback);
     };
+    /**
+     * Report the error to Ingest api service and show case on DDMetrics
+     * @param error 
+     */
+    reportErrorToIngestApi(error) {
```

Comment:
Not sure how this is reporting the error to ingest. Are we handling this through a collector status? 

Alos shouldn't this have some sort of callback? arent we going to have a race condition with the async callback?

like this https://github.com/alertlogic/paws-collector/blob/6a6dcc561aa2bee5fa9085ed4ab3016637ba8c89/paws_collector.js#L387-L389